### PR TITLE
docs: update to latests docs design

### PR DESCRIPTION
**Pull request by HubbeDev**
This PR should fix most of #619. Let me know if I missed something.

There are still some card examples left. I could take a look at them later this week.

### CODE REVIEW

**Review by huntabyte — CHANGES_REQUESTED** [review]
Incredible job! Just a couple minor things!

File: apps/www/src/lib/components/docs/nav/docs-sidebar-nav-items.svelte [hunk 1]
```diff
@@ -11,9 +11,10 @@
 		{#each items as item, index (index)}
 			{#if item.href}
 				<a
+					data-sveltekit-keepfocus
```
[the remark this answers is not in the document]
Comment (HubbeDev) [reply apps/www/src/lib/components/docs/nav/docs-sidebar-nav-items.svelte:14]: Aha. I may have misread the docs about it. I was trying to implement the same behavior as when navigating on ui.shadcn.com where you tab directly after clicking a link. The focus will continue where you click. But now when I think about it. That doesn't sound like a good idea in an assessability perspective

**Comment by HubbeDev** [comment]
@huntabyte Do you think the Geist font weights should be preloaded now when self hosted? 
I think we maby should atleast preload the "regular" and "semibold" becaouse we use those the most. What do you think?

Im not 100% sure tho whats the best way to preload fonts in sveltekit. Is it with svelte:head?

**Comment by HubbeDev** [comment]
@huntabyte I will check! The fonts that is in use the most should be preloaded to prevent font flickering & extra loadtimes. But not all of them if loading many. But i will try to find more information about how to do it correctly in sveltekit.

**Comment by HubbeDev** [comment]
@huntabyte Last commit added the calendar component to the theme customizer. But for some reason, the calendar is overflowing in the default theme. I'm not really sure what is causing that. Will look into it.

**Review by huntabyte — APPROVED** [review]
Great work, @HubbeDev you're the man!

**Pull request merged**